### ISSUE TITLE
Tags are not showing on the Folder nor File

### ISSUE THREAD

**Issue by wawa2016** [ciur/papermerge#633]
**Description**
I can create colored Tags but after I assign them to a folder or a File they are not showing in the web interface.

Could kindly please suggest how to resolve it?

**Info:**
 - Papermerge Version: 3.2
[papermerge-container-log.txt](https://github.com/user-attachments/files/16826141/papermerge-container-log.txt)
[docker-compose-yml.txt](https://github.com/user-attachments/files/16826142/docker-compose-yml.txt)

**Comment by Kimbaras** [comment]
I believe this is a general bug with 3.x as I went back as far as 3.0.3 and the issue seem present on each version (3.2, 3.1 and 3.0.3 tested).

Thanks

**Comment by ciur** [comment]
Do you mean creating tags in "tags" tab? (The menu on the left side). 

Does problem happen in 3.3? Could you attach a video recording reproducing the problem ?

**Comment by Kimbaras** [comment]
Nope, creating tags from the dedicated tab seems to be working (even if the UI seems buggy, screen attached 
![tag_ui](https://github.com/user-attachments/assets/f688e156-1866-4bc2-ac07-53b128187750)
) as I have the option to use them while editing folders / files.

The issue here (for me and the original poster) is that tags are not shown anywhere AFTER having them applied to something.
I'll try to record a video of the issue in the next days.

Regaring the question about 3.3... can't say as from the little tests I've done today it's barely usable, the main thing that stopped me is the folder creation UI being completely off-center (screen 
![ui_33](https://github.com/user-attachments/assets/aafc5f34-9644-44a7-9e1c-28ee09eb5312)
).
But this is off topic of the main issue, if you want I can open a dedicated issue for that.

**Comment by ciur** [comment]
The issue with "UI off-center of the screen" was fixed in 3.3.1.

Would you give 3.3.1 another try ?

**Comment by Kimbaras** [comment]
Hey, 

From my side, on 3.3.1 both issue (the off-center UI and the tags not showing) seems fixed!

![Image](https://github.com/user-attachments/assets/f6068298-5fca-45b0-8bdf-34628928c613)
![Image](https://github.com/user-attachments/assets/8e4d5f98-8d7b-40df-841d-7113e9a83b64)

Thanks!
